docs/manual/java/guide/build/EclipseMavenInt.md

```diff
@@ -0,0 +1,31 @@
+# Importing an existing Maven project into Eclipse
+
```

Comment:
Maybe this page should also include the "Check prerequisites" section from `EclipseMavenNew.md`.